pkg/client/cache/fifo.go

```diff
@@ -241,6 +241,17 @@ func (f *FIFO) Replace(list []interface{}, resourceVersion string) error {
 	return nil
 }
 
+// Resync will touch all objects to put them into the processing queue
+func (f *FIFO) Resync() error {
+	f.lock.Lock()
+	defer f.lock.Unlock()
+	for id := range f.items {
+		f.queue = append(f.queue, id)
+	}
+	f.cond.Broadcast()
```

Comment:
don't understand how this implementation is useful. same thing as the resync with the k8s-mesos historical FIFO: here the `items` map only has added/updated items that are **already** in the queue. this func adds them again to the queue. i don't understand what good this does anyone to have redundant items in the queue. i think i'm still missing something